playground/README.md

```diff
@@ -7,3 +7,4 @@ As any playground - we have a few rules:
 * Add content and commit messages in english
 * Try to add descriptive commit messages
 * Be nice with any other content you find in here
+* Have fun
```

Comment:
Nice

Reply:
sad af

Reply:
Review is not clear - how do you propose to fix this?

Reply:
Having a beer is always a good start. Except for driving, #stay-safe.